### PR TITLE
replacing sysadmin with new config

### PR DESCRIPTION
**Pull request by HamzaIbnFarooq**
### Related tickets
https://github.com/mitodl/edx-sysadmin/issues/1

### What this PR does
replace sysadmin navigation w.r.t edx-sysadmin plugin

### NOTE
This is a draft PR and will be edited later to create an upstream PR

### CODE REVIEW

File: common/djangoapps/util/sysadmin_utils.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    Checks if user has access to sysadmin panel or not
+    :param user: User object of currently loggedin user
+    :return boolean: True if user has access to syadmin else False
+    """
+    # TODO: Give access to Course Admins
+    if user and user.is_staff:
+        return True
+
+
+def is_edx_sysadmin_enabled():
```
Comment (ziafazal) [inline common/djangoapps/util/sysadmin_utils.py]: Do we need this as per discussion we are not going to attach sysadmin with `ENABLE_SYSADMIN_DASHBOARD` feature flag.

File: common/djangoapps/util/sysadmin_utils.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+
+from django.conf import settings
+
+from edx_django_utils.plugins import get_plugin_apps
+from openedx.core.djangoapps.plugins.constants import ProjectType
+
```
Comment (ziafazal) [inline common/djangoapps/util/sysadmin_utils.py:9]: define a method `show_sysadmin_dashboard` here like this 
```
def show_sysadmin_dashboard(user):
 return is_edx_sysadmin_enabled() and is_edx_sysadmin_installed() and user_has_access_to_sysadmin(user)
```
use that method where we need to determine visibility of sys admin

File: common/djangoapps/util/sysadmin_utils.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+def is_edx_sysadmin_installed():
+    """
+    Checks if edx-sysadmin plugin is installed in the platform
+    :return boolean: True if plugin is installed else False
+    """
+    for plugin in get_plugin_apps(ProjectType.LMS):
```
Comment (ziafazal) [inline common/djangoapps/util/sysadmin_utils.py]: I think [PluginManager's](https://github.com/edx/edx-django-utils/blob/71f239156bbbaa2a5841eba55029433b14952b90/edx_django_utils/plugins/plugin_manager.py#L42) `get_plugin` method is better way of getting an installed plugin

**Review by ziafazal — CHANGES_REQUESTED** [review]
@HamzaIbnFarooq couple of suggestions and a question.

File: common/djangoapps/util/sysadmin_utils.py (hunk 1 above)
Comment (HamzaIbnFarooq) [reply common/djangoapps/util/sysadmin_utils.py]: I have removed ENABLE_SYSADMIN_DASHBOARD feature flag now.

File: common/djangoapps/util/sysadmin_utils.py (hunk 2 above)
Comment (HamzaIbnFarooq) [reply common/djangoapps/util/sysadmin_utils.py:9]: updated.

File: common/djangoapps/util/sysadmin_utils.py (hunk 3 above)
Comment (HamzaIbnFarooq) [reply common/djangoapps/util/sysadmin_utils.py]: Converted to that.

**Review by ziafazal — APPROVED** [review]
👍

**Comment by HamzaIbnFarooq** [comment]
This PR has been moved to https://github.com/mitodl/edx-platform/pull/252

**Pull request closed without merging**